startup_runners/startup_runner_runner.c

```diff
@@ -0,0 +1,38 @@
+/*
+ * Iterations runner for C benchmarks.
+ *
+ * Code style here is KNF, but with 4 spaces instead of tabs.
+ */
+
+/* To correctly expose asprintf() on Linux */
+#define _GNU_SOURCE
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <limits.h>
+#include <errno.h>
+#include <string.h>
+#include <dlfcn.h>
```

Comment:
Some of these includes are redundant. Certainly `dlfcn.h`, maybe `limits.h` and `string.h`.